documentation/docs/getting_started/java.mdx

```diff
@@ -11,62 +11,25 @@ keywords:
 ---
 import WarningPasswordStorage from '../_admonitions/_password_storage.md';
 
-# Getting Started with Java
+# IOTA Client Java Library
+
+Get started with the official IOTA Client Java Library.
 
 ## Security
 
 <WarningPasswordStorage/>
 
 ## Requirements
 
-* The latest [Java Development Kit (JDK)](https://www.oracle.com/java/technologies/downloads/).
```

Comment:
Why are we removing all this ?

Reply:
Not needed anymore when installing with Maven.
Building everything from source however would require more requirements etc. But I guess we should do that in a separate doc file. Same for wallet.